### ISSUE TITLE
Question: Updating managed servers through centralised Cockpit

### ISSUE THREAD

**Issue by abrousseau001** [cockpit-project/cockpit#12216]
This is more of a question on how to get this done/if it is possible, but I was trying to add servers to the cockpit-dashboard in order to have a single cockpit instance in which I could manage multiple servers. During my test I signed in to an Ubuntu 18.04 that had cockpit along with cockpit-packagekit installed. Since this is an Ubuntu server the root account is not normally enabled so I signed in with the account with sudo permissions. When I did this I could look at the user I signed in as to see it was system-admin (see below).

![account](https://user-images.githubusercontent.com/11802189/60602998-e7b1a880-9d82-11e9-9d3e-df6fafa9044c.png)

![basic_info](https://user-images.githubusercontent.com/11802189/60602948-cc469d80-9d82-11e9-8015-9046ae0e7db8.png)

When I go to software updates it states that it failed to obtain authentication. I get a log entry that it is trying to login using the user root which is the main cockpit's user not the user specified to connect to the system

![failed](https://user-images.githubusercontent.com/11802189/60602964-d49ed880-9d82-11e9-9430-74d63eaa5cfa.png)

![login_error](https://user-images.githubusercontent.com/11802189/60602976-d9638c80-9d82-11e9-8ca8-a90b9ef4ad81.png)

Any assistance to why the user being provided is not the same as the user that was required to login to the managed cockpit server would be most appreciated.

**Comment by marusak** [comment]
This looks very much like when you haven't checked `Reuse my password for privileged tasks` on login.
![this](https://user-images.githubusercontent.com/12330670/60604977-fadf6b00-9db8-11e9-8049-0559f56093f0.png)
Also while logged-in you should see in top right corner (left of user name) `privileged`.

**Comment by abrousseau001** [comment]
I did check reuse my password for privileged actions, but I am running cockpit on my local system, using the dashboard to login to a different system I want to manage. I am prompted for the credentials to log into that system, but the privileged action should be taken with the login I provided to access the other system, not the one I have logged into locally.

**Comment by xruth** [comment]
@abrousseau001 I ran into the exact same issue as you did. Once I disabled the root user on the remote system (went to cockpit directly on the remote host and checked the box "lock account" under the root user's settings), the updates started to show up on my local installation. Did not try to install though. 
I am still missing the option to reuse the password for privileges when adding remote systems. Is there a way to achieve this?
